### PR TITLE
Allow to set timeout in client

### PR DESCRIPTION
**Pull request by RadekMolenda**
Allow timeout options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,18 @@ $integration = $bearer->integration('integration_id');
...
 ```
 
+### Setting the request timeout
+
+By default bearer client request and connection timeouts are set to 5 seconds. Bearer allows to increase the request timeout to up to 30 seconds
+
+``` php
+$bearer = new Bearer\Client('BEARER_SECRET_KEY');
+
+$integration = $bearer->integration('integration_id', 20, 10); // set timeout to 20 seconds and connectTimeout to 10 seconds
```
Comment (RadekMolenda) [inline README.md]:
```suggestion
$integration = $bearer->integration('integration_id');
```

File: README.md [hunk 2]
```diff
@@ -73,6 +73,18 @@ $integration = $bearer->integration('integration_id');
 $integration->invoke('functionName');
 ```
 
+### Setting the request timeout
+
+By default bearer client request and connection timeouts are set to 5 seconds. Bearer allows to increase the request timeout to up to 30 seconds
+
+``` php
+$bearer = new Bearer\Client('BEARER_SECRET_KEY');
```
Comment (RadekMolenda) [inline README.md]:
```suggestion
$bearer = new Bearer\Client('BEARER_SECRET_KEY', 20, 10); // set timeout to 20 seconds and connectTimeout to 10 seconds
```

File: README.md [hunk 3]
```diff
@@ -64,13 +64,16 @@ $github
     ->request('PUT', '/user/starred/bearer/bearer', [ "headers" => [ "Content-Length" => 0 ] ]);
 ```
 
-## Calling custom functions
+### Setting the request timeout
 
-```php
-$bearer = new Bearer\Client('BEARER_SECRET_KEY');
+By default bearer client request and connection timeouts are set to 5 seconds. Bearer allows to increase the request timeout to up to 30 seconds
```
Comment (cfabianski) [inline README.md]:
```suggestion
By default, Bearer client request and connection timeouts are set to 5 seconds. Bearer allows to increase the request timeout to up to 30 seconds
```

**Comment by RadekMolenda** [comment]
https://github.com/Bearer/bug-tracker/issues/3

**Pull request merged**